tests/StatsdClient.Tests/CommandIntegrationTests.cs

```diff
@@ -92,398 +94,345 @@ public void counter_sample_rate()
             AssertWasReceived("histogram:42.1|h");
         }
 
         [Test]
         public void histogram_double_tags()
         {
-            DogStatsd.Histogram("histogram", 42.1, tags: new[] { "tag1:true,tag2" });
+            _dogStatsdService.Histogram("histogram", 42.1, tags: new[] { "tag1:true,tag2" });
             AssertWasReceived("histogram:42.1|h|#tag1:true,tag2");
         }
 
         [Test]
         public void histogram_double_sample_rate()
         {
-            DogStatsd.Histogram("histogram", 42.1, 1.1);
+            _dogStatsdService.Histogram("histogram", 42.1, 1.1);
             AssertWasReceived("histogram:42.1|h|@1.1");
         }
 
         [Test]
         public void histogram_double_sample_rate_tags()
         {
-            DogStatsd.Histogram("histogram", 42.1, sampleRate: 1.1, tags: new[] { "tag1:true,tag2" });
+            _dogStatsdService.Histogram("histogram", 42.1, sampleRate: 1.1, tags: new[] { "tag1:true,tag2" });
             AssertWasReceived("histogram:42.1|h|@1.1|#tag1:true,tag2");
         }
 
-        [Test]
-        public void distribution()
-        {
-            DogStatsd.Distribution("distribution", 42);
-            AssertWasReceived("distribution:42|d");
-        }
-
-        [Test]
-        public void distribution_tags()
-        {
-            DogStatsd.Distribution("distribution", 42, tags: new[] { "tag1:true", "tag2" });
-            AssertWasReceived("distribution:42|d|#tag1:true,tag2");
-        }
-
-        [Test]
-        public void distribution_sample_rate()
-        {
-            DogStatsd.Distribution("distribution", 42, sampleRate: 1.1);
-            AssertWasReceived("distribution:42|d|@1.1");
-        }
-
-        [Test]
-        public void distribution_sample_rate_tags()
-        {
-            DogStatsd.Distribution("distribution", 42, sampleRate: 1.1, tags: new[] { "tag1:true,tag2" });
-            AssertWasReceived("distribution:42|d|@1.1|#tag1:true,tag2");
-        }
+       
 
-        [Test]
-        public void distribution_double()
-        {
-            DogStatsd.Distribution("distribution", 42.1);
-            AssertWasReceived("distribution:42.1|d");
-        }
-
-        [Test]
-        public void distribution_double_tags()
-        {
-            DogStatsd.Distribution("distribution", 42.1, tags: new[] { "tag1:true,tag2" });
-            AssertWasReceived("distribution:42.1|d|#tag1:true,tag2");
-        }
-
-        [Test]
-        public void distribution_double_sample_rate()
-        {
-            DogStatsd.Distribution("distribution", 42.1, 1.1);
-            AssertWasReceived("distribution:42.1|d|@1.1");
-        }
-
-        [Test]
-        public void distribution_double_sample_rate_tags()
-        {
-            DogStatsd.Distribution("distribution", 42.1, sampleRate: 1.1, tags: new[] { "tag1:true,tag2" });
-            AssertWasReceived("distribution:42.1|d|@1.1|#tag1:true,tag2");
-        }
```

Comment:
I think those were accidentally removed. Can we add them back?

Reply:
No, I moved them to this class 

DogstatsdStaticApiTests

because you are testing a static API, you can't just have the tests in a class like this because as soon as you have a multithreaded test runner such as NCrunch, then the config gets called twice which is something we don't want to support any more.

See DogstatsdStaticApiTests and in there you can see that the config will only get called once by using the NUnit attributes

Reply:
I understand. 
We have a 1-1 mapping between the tests in `CommandIntegrationTests.cs` and `DogstatsdStaticApiTests ` for other resources, so I think it'd make sense to have similar tests here for "Distribution" resources  (using an instance of DogStatsD instead of the static API).

Reply:
Latest commits move the tests to the CommandIntegrationTests